go.mod

```diff
@@ -2,6 +2,104 @@ module github.com/influxdata/influxdb/v2
 
 go 1.17
 
+require (
```

Comment:
I'm not sure either, but I've seen it before since the 1.17 upgrade. I've been merging them back together and then `go mod tidy`-ing to re-sort, not sure if it's worth continuing to fight it